src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -526,10 +525,10 @@
 	gap: 6px;
 }
 
+.interactive-response-progress-tree .monaco-list,
```

Comment:
Does it matter whether these are one the monaco-list or the parent? Shouldn't this just match used-context-list exactly?

Reply:
I seem to recall that it was important to set it on the monaco-list, can't remember why though and it looks fine setting it on the parent element for now